tpm2tools/misc_test.go

```diff
@@ -0,0 +1,43 @@
+package tpm2tools
```

Comment:
These two tests should be [table-driven](https://github.com/golang/go/wiki/TableDrivenTests) to prevent duplication.

Reply:
Done.